formulas/helpers.js

```diff
@@ -179,6 +179,46 @@ class FormulaHelpers {
     return number;
   }
 
+  equalOP(input1, input2) {
+    if(!Array.isArray(input1) && !Array.isArray(input2)){
+      return input1 === input2
+    }
+    const rv = []
+    
+    if(Array.isArray(input1) && Array.isArray(input2)){
+      let maxRow = Math.max(input1.length, input2.length);
+      for(let row = 0; row < maxRow; row++){
+        let row1 = (input1.length > row) ? input1[row] : []
+        let row2 = (input2.length > row) ? input2[row] : []
+        let curr = []
+        let maxCol = Math.max(row1.length, row2.length);
+        for(let col = 0; col < maxCol; col++){
+          let item1 = (row1.length > col) ? row1[col] : null
+          let item2 = (row2.length > col) ? row2[col] : null
+          curr.push(item1 === item2)
+        }
+        rv.push(curr)
+      }
+      return rv
+    }
+
+    let array = (Array.isArray(input1)) ? input1 : input2;
+    let constant = (Array.isArray(input1)) ? input2 : input1;
+
+    //for a single value
+    if(array.length === 1 && array[0].length === 1){
```

Comment:
Why do we need to special case this?

Reply:
A lot of the current implementations of formulas rely on this comparison. I can change them locally if prefered